configs/golden.cfg

```diff
@@ -0,0 +1,11 @@
+{
+    "hostname"  : "golden",
+    "node_coord" : "golden_custom",
+    "repo_mirror" : "http://seedy.us.rdlabs.hpecorp.net",
+    "repo_release" : "jessie",
+    "repo_areas" : ("main", "contrib", "non-free"),
+    "other_mirrors" : "deb http://hlinux-deejay.us.rdlabs.hpecorp.net/l4fame/ catapult main contrib non-free",
```

Comment:
'deb [trusted=yes] https://downloads.linux.hpe.com/repo/l4fame/Debian/ testing main'

Reply:
No. Still missing packages (at least tm-node..).

Reply:
that's because it's called "l4fame-node" at the moment:

http://downloads.linux.hpe.com/SDR/repo/l4fame/Debian/pool/main/l/l4fame-node/

Reply:
no "tm-zbridge" package.

Reply:
No, there's not.  Where is it referenced?